### PR TITLE
media-video/subliminal: verbump to 2.0.3, remove old

### PR DESCRIPTION
**Pull request by Coacher**

### CODE REVIEW

**Comment by Coacher** [comment]
CC @gentoo/proxy-maint

**Comment by monsieurp** [comment]
@gentoo/proxy-maint

File: media-video/subliminal/subliminal-2.0.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4} )
```
Comment (gktrk) [inline media-video/subliminal/subliminal-2.0.3.ebuild]: [setup.py](https://github.com/Diaoul/subliminal/blob/master/setup.py#L65) claims support for python3_5. should we add it to the list?

**Comment by gktrk** [comment]
I don't know why it says I commented on an outdated diff. Anyway, [setup.py](https://github.com/Diaoul/subliminal/blob/master/setup.py#L65) claims support for python3_5. should we add it to the list?

**Comment by Coacher** [comment]
If you can buildtest and runtest subliminal with Py 3.5, then feel free to add it, but in a separate commit please. Until recently Py 3.5 on Gentoo was affected by bug 585060. Now it's fixed and I am planning to gradually add Py 3.5 support to all my python packages, but I don't mind if you beat me to it.

**Comment by Coacher** [comment]
Thank you very much, @monsieurp.

**Pull request closed without merging**